VAMobile/src/store/api/demo/store.ts

```diff
@@ -127,6 +127,7 @@ export const initDemoStore = async (): Promise<void> => {
     import('./mocks/payments.json'),
     import('./mocks/prescriptions.json'),
     import('./mocks/notifications.json'),
+    import('../../../api/demoMocks/getAuthorizedServices.json'),
```

Comment:
Does getAuthorizedServices.json need to be in a new demoMocks directory instead of just being in demo/mocks like the other mocks?

Reply:
we are moving all the demo mode mocks when updating the api's to react query to this new folder so we can make sure data is mocked properly and include things that weren't there before like cerner facilities in the get facility info ticket

Reply:
Depending on our plans for demo mode with React Query, it might make sense to keep the mock data in `store/api/demo/mocks` if we're going to continue using the Redux store for demo mode. That way, all the demo mode related resources are in the same location. It might be confusing if the data that the demo store consumes is in another part of the repo. I have a [ticket](https://github.com/department-of-veterans-affairs/va-mobile-app/issues/6798) for investigating other ways of handling demo with React Query, so that could determine whether we'll still use the demo store, and where the mock data should ultimately live

Reply:
I will agree with that, lets use that ticket to determine that and move things as needed and we can just continue as is while we are updating the api's until that one is done